source/Ocl/IOclConverter.cs

```diff
@@ -1,12 +1,13 @@
 using System;
 using System.Collections.Generic;
+using System.Reflection;
 
 namespace Octopus.Ocl
 {
     public interface IOclConverter
     {
         bool CanConvert(Type type);
-        IEnumerable<IOclElement> ToElements(OclConversionContext context, string name, object value);
+        IEnumerable<IOclElement> ToElements(OclConversionContext context, PropertyInfo? propertyInfo, object value);
```

Comment:
I'm not a huge fan of this interface change. I want to love it but I'm just not finding my way there. 

What would this mean for converters used at the root-level of a document? It would be nice if this were part of the context for this conversion. I can see potential need in the future for other context-specific info like the parent context.